bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: Add SelectSSOProvider flag/Add rh clientSecret/id

## Description
Adding a flag in order to be able to select which so provider to choose from i.e `redhat_sso` or `mas_sso`. Also adding the env vars for ClientIDFile and ClientSecretFile to the service template as well.

Jira: [MGDSTRM-8210](https://issues.redhat.com/browse/MGDSTRM-8210)

## Verification Steps
1. Create cluster.
2. Deploy kas-fleet-manager to cluster. 
3. Run the command `make deploy/service IMAGE_TAG=<your-image-tag-here> SSO_PROVIDER_TYPE=<provider>` when deploying kas-fleet-manager to the cluster
4. Use this endpoint  [here](https://api.openshift.com/?urls.primaryName=kafka%20service%20fleet%20manager%20service#/security/getSsoProviders) to test and ensure the provider is being changed.

Supporting docs : [Here](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/docs/deploying-kas-fleet-manager-to-openshift.md)

## Checklist (Definition of Done)
